### ISSUE TITLE
Excise quippy

### ISSUE THREAD

**Issue by gabor1** [libAtoms/silicon-testing-framework#1]
In order to help people adopt this, it would be good to excise quippy from tests and potentials as much as possible. More important to excise it from tests.

**Comment by noambernstein** [comment]
Keep in mind that this is not the new testing framework, just a slightly
cleaned up version of the old Si framework for release with the paper,
right?  My new refactored framework is separate, and I'm not even sure I
gave you a copy.  It's certainly not named "silicon-testing-framework".  We
_still_ need to set a time to meet with anyone who might be interested to
deal with that.

**Comment by gabor1** [comment]
Yes, I temporarily forgot - the GitHub version looked very clean :-) 

I am wanting to use it for other things now, and eventually start to push
others to use it too, e.g. I talked to Aidan Thompson about it. 

Can you share your refactored version ? How were you thinking of releasing it? 

G

**Comment by noambernstein** [comment]
On Sat, Jul 14, 2018 at 7:24 PM gabor1 <notifications@github.com> wrote:

> Yes, I temporarily forgot - the GitHub version looked very clean :-)
>
> I am wanting to use it for other things now, and eventually start to push
> others to use it too, e.g. I talked to Aidan Thompson about it.
>
> Can you share your refactored version ? How were you thinking of releasing
> it?
>

I could release what I have at any time, but my sense is that it's probably
still going to need a lot of changes, so any serious changes I make would
then have to be released later.  Most sensible thing is probably to share
with you + 1-2 more people as a collaboration, and once it's stable then do
a real release.

          Noam

>

**Comment by gabor1** [comment]
Repo on gc121mac1?

-- Gábor
